mikeio/spatial/grid_geometry.py

```diff
@@ -296,6 +308,22 @@ class Grid2D(_Geometry):
     _orientation: float
     is_spectral: bool
 
+    def __eq__(self, other):
```

Comment:
right, we should probably remove it again. It's just that one of the dfs3 tests failed due to _origin:

![image](https://user-images.githubusercontent.com/34088801/218836749-ba14be4f-77bb-4580-8075-bc7384b2ea8f.png)

Reply:
@ecomodeller , do you know why this happens in this case?

Reply:
No, I'll try to find out, useful to understand.

Reply:
I supppose this error is raised because, even though we claim with type hints that origin is a tuple of two floats, at runtime it is a `np.ndarray` which doesn't implement an `__eq__` method that adhere to the behaviour that dataclasses expect.

And even if it did, the numbers are not identical due to rounding errors.

So the question is, is that rounding error (presumably caused by conversion from single to double precision) a bug or a feature?

```python
>>> ds.geometry.origin
array([ 586422.9462332, 6142644.1548212])
>>> dsnew.geometry.origin
array([ 586422.9462332, 6142644.1548213])
>>> ds.geometry.origin == dsnew.geometry.origin
array([ True, False])
```